docs-src/concepts/what-is-the-temporal-web-ui.md

```diff
@@ -9,7 +9,7 @@ tags:
 ---
 
 The Temporal Web UI provides users with Workflow Execution state and metadata for debugging purposes.
-It ships with every [Temporal CLI](/concepts/what-is-the-temporal-cli) release and [Docker Compose](/kb/all-the-ways-to-run-a-cluster#docker--docker-compose) update and is available with [Temporal Cloud](/cloud).
```

Comment:
`/concepts/what-is-the-temporal-cli` is actually the correct link -
Link magic converts it.